x-pack/test/security_solution_cypress/es_archives/auditbeat/data.json

```diff
@@ -47,14 +47,14 @@
         "name" : "siem-kibana"
       },
       "agent" : {
-        "type" : "auditbeat",
```

Comment:
why the changes here? This is an auditbeat file?

Reply:
analyzer entity api will result in 0 hits due to the schema. i could add a different event or something so the test i added works, but since nothing here is concerned about agent type or event module, this seemed ok

Reply:
Could this have broken the tests?

Reply:
ya the overview ones for sure, needs to be agent.type: 'winlogbeat' and event.module: 'sysmon' for the analyzer api to work. just going to add a second event in this archive to keep the tests simple.